content/blog/multitenancy-for-kubernetes-clusters-using-hpe-alletra-6000-and-nimble-storage.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Multitenancy for Kubernetes clusters using HPE Alletra 6000 and Nimble Storage
+date: 2021-06-15T15:00:00.000Z
+author: Michael Mattsson
+authorimage: /img/portrait-2020-192px.png
+tags:
+  - hpe-nimble-storage
+  - hpe-3par-and-primera
+---
+In a storage infrastructure economy where IT is under constant pressure to deliver more with less, yet still provide a high standard in data services directly to end users without compromising system or data security.
+
+With the introduction of [HPE CSI Driver for Kubernetes 2.0](https://community.hpe.com/t5/Around-the-Storage-Block/HPE-CSI-Driver-for-Kubernetes-now-available-for-HPE-Alletra/ba-p/7136280) and the software powering HPE Alletra 6000 and Nimble Storage, Hewlett Packard Enterprise introduces multitenancy for Kubernetes clusters accessing persistent volumes on the aforementioned storage arrays.
+
+The term "tenant" is ambigous within the industry. For HPE Alletra 6000 and Nimble Storage, a tenant is a storage appliance user account with confined privileges to volumes existing within one or many "folders" on the array defined by a storage administrator. In turn, a folder is a logical construct to losely group volumes together which allows capacity and performance accounting to be limited on per folder level. The tenant may manage any aspect of the volume within the folder and may not exceed the boundaries set on the folder.
```

Comment:
Remove quotes around "tenant" and italicize it instead. Same with "folders".

Second sentence, change "losely" to "loosely" and add a comma after "volumes together" so it reads: "...to loosely group volumes together, which allows capacity..."